yt/data_objects/tests/test_profiles.py

```diff
@@ -356,39 +356,39 @@ def tearDown(self):
     def test_unequal_data_shape_profile(self):
         density = np.random.random(128)
         temperature = np.random.random(128)
-        cell_mass = np.random.random((128, 128))
+        mass = np.random.random((128, 128))
 
         my_data = {
-            "density": density,
-            "temperature": temperature,
-            "cell_mass": cell_mass}
+            ('gas', 'density'): density,
+            ('gas', 'temperature'): temperature,
+            ('gas', 'mass'): mass}
         fake_ds_med = {"current_time": yt.YTQuantity(10, "Myr")}
         yt.save_as_dataset(fake_ds_med, "mydata.h5", my_data)
 
         ds = yt.load('mydata.h5')
 
         assert_raises(
             YTProfileDataShape,
-            yt.PhasePlot, ds.data, 'temperature', 'density', 'cell_mass')
+            yt.PhasePlot, ds.data, 'temperature', 'density', 'mass')
```

Comment:
This will conflict with #3101 and #3194
```suggestion
            yt.PhasePlot, ds.data, ("gas", "temperature"), ("gas", "density"), ("gas", "mass"))
```

Reply:
Still think this is going to cause the yt data tests to fail.

Reply:
This has been changed.